src/r_data.c

```diff
@@ -391,6 +391,9 @@ static void R_GenerateLookup(int texnum, int *const errors)
 	  int x, x1 = patch++->originx, x2 = x1 + SHORT(realpatch->width);
 	  const int *cofs = realpatch->columnofs - x1;
 	  
+	  if (!R_IsPatchLump(pat))
+	    continue;
```

Comment:
I'd like to report texture, patch name and number to stdout in this case.

Reply:
Ah, sorry, but stderr makes more sense here. Also, could you print the function name instead of "Warning"?